### PR TITLE
MWPW-156262 use standard tab browsing in collection

### PR DESCRIPTION
**Pull request by npeltier**
- remove custom key handling for accessibility in merch cards,
- in case cards are added within a collection, add tabindex so focus order follows style order,

note that now only TAB & SHIFT+TAB is available now as it is in other places. 
Also note a separate PR will be made to flip order of sidenav & collection not to have to browse through the entire collection to get to the filters

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.live/?martech=off
- After: https://MWPW-156262--milo--npeltier.hlx.live/?martech=off

***with merch-card-collection in catalog***
- Before: https://main--cc--adobecom.hlx.live/products/catalog?martech=off
- After: https://main--cc--adobecom.hlx.live/products/catalog?martech=off&milolibs=MWPW-156262--milo--npeltier

***with merch-card-collection on special offers page***
- Before: https://main--cc--adobecom.hlx.live/products/special-offers?martech=off
- After: https://main--cc--adobecom.hlx.live/products/special-offers?martech=off&milolibs=MWPW-156262--milo--npeltier

### CODE REVIEW

**Comment by yesil** [comment]
@npeltier when we leave the last card, the focus ring is not removed from the card.

File: libs/blocks/merch-card-collection/merch-card-collection.js [hunk 1]
```diff
@@ -162,6 +162,7 @@ async function initMerchCards(filtered, preferences, cardsRoot) {
   // re-order cards, update card filters
   [...cardsRoot.children].filter((card) => card.tagName === 'MERCH-CARD').forEach((merchCard) => {
     merchCard.style.display = 'none';
+    merchCard.keyHandling = false;
```
Comment (yesil) [inline libs/blocks/merch-card-collection/merch-card-collection.js:165]: this seems to be breaking navigation by arrow keys in the catalog page.

Comment (npeltier) [reply libs/blocks/merch-card-collection/merch-card-collection.js:165]: yes, it just brings navigation back to standard TAB browsing, no custom code at all, (checked with accessibility expert on ticket that seemed ok with it), plus former catalog page didn't have it. Is arrow key browsing for catalog pages (and collections) a requirement from someone else?

**Comment by npeltier** [comment]
after discussion with @yesil, we removed arrow browsing, leaving everything to default browser accessibility management

**Review by 3ch023 — APPROVED** [review]

**Comment by Roycethan** [comment]
@npeltier / @yesil  As shown below in tab order arrangement - 
Order 12 - focus is not on the product icon, slightly down
Order 14 -  focus/ tabbing here is not on any elements. either it shouldn't be there or should be on a element if exists.
image

**Comment by Roycethan** [comment]
Tracked above issues at https://jira.corp.adobe.com/browse/MWPW-156875 and making this Ready for stage

**Pull request merged**